src/borg/repository.py

```diff
@@ -843,6 +850,8 @@ def report_error(msg):
             transaction_id = self.get_transaction_id()
             current_index = self.open_index(transaction_id)
             logger.debug('Read committed index of transaction %d', transaction_id)
+        except self.AtticRepository as exc:
+            raise
```

Comment:
This is even more unclear to me. What do you want this to look like? Anyway, even if we decide to keep the simple `raise` here, the `as exc` part can go so this hunk will change in a fix-up commit.